lib/rubocop/cop/rspec/leaky_constant_declaration.rb

```diff
@@ -23,35 +23,51 @@ module RSpec
       #   # bad
       #   describe SomeClass do
       #     OtherClass = Struct.new
-      #     CONSTANT_HERE = 'is also denied'
+      #     CONSTANT_HERE = 'I leak into global namespace'
       #   end
       #
       #   # good
       #   describe SomeClass do
       #     before do
       #       stub_const('OtherClass', Struct.new)
-      #       stub_const('CONSTANT_HERE', 'is also denied')
+      #       stub_const('CONSTANT_HERE', 'I am undefined after the example')
```

Comment:
Maybe this should say "I am removed after the example." or "I only exist during this example." or something like that.

The word "undefined" makes me think of UB in C or something I generally need to be concerned about.